src/mmw/apps/home/views.py

```diff
@@ -115,7 +115,8 @@ def get_client_settings(request):
             'base_layers': get_layer_config('basemap'),
             'stream_layers': get_layer_config('stream'),
             'boundary_layers': get_layer_config('boundary'),
-            'overlay_layers': get_layer_config('overlay'),
+            'vector_layers': get_layer_config('vector'),
+            'raster_layers': get_layer_config('raster'),
```

Comment:
Consider adding the word "overlay" in the attribute names just so it's clear that's what they are.